src/commonTest/kotlin/org/cru/godtools/tool/internal/Channel.kt

```diff
@@ -0,0 +1,6 @@
+package org.cru.godtools.tool.internal
+
+import kotlinx.coroutines.channels.Channel
+import kotlinx.coroutines.withTimeout
+
+suspend inline fun <T : Any?> Channel<T>.receive(timeout: Long) = withTimeout(timeout) { receive() }
```

Comment:
What does this Channel do?

Reply:
I'm just adding a timeout parameter to `receive()` for tests. that way misbehaving tests don't hang indefinitely and will error out with a timeout instead.

https://kotlinlang.org/docs/channels.html